tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Features/Organisations GetServiceRecipients.feature

```diff
@@ -1,7 +1,7 @@
-﻿Feature: Display all of the Service Recipients for an Organisation
-	As a Buyer
-	I want to be able to view all of the Service Recipients for a organisation
-	So that I can ensure all of the information is correct
+﻿Feature: Display all of the Service Recipients for an organisation
+    As a Buyer
+    I want to be able to view all of the Service Recipients for a organisation
```

Comment:
Sorry, should be *an* here too – must've missed it last time.

Reply:
👍